packages/message-box/message-box.scss

```diff
@@ -0,0 +1,45 @@
+@import "~@fremtind/jkl-core/build/scss/variables/colors.scss";
+@import "~@fremtind/jkl-core/build/scss/variables/spacing.scss";
+@import "~@fremtind/jkl-core/build/scss/mixins/utils.scss";
+@import "~@fremtind/jkl-core/build/scss/typography/headings.scss";
+@import "~@fremtind/jkl-core/build/scss/typography/paragraphs.scss";
+
+@mixin colored-line($color) {
+    @include position(absolute);
+    content: " ";
+    height: 100%;
+    width: $component-spacing--xs;
+    background-color: $color;
+}
+
+@mixin box($bg-color) {
+    position: relative;
+    background-color: $bg-color;
+    width: calc(3 *#{$layout-spacing--xxl});
+    min-height: $layout-spacing--xl;
+    padding: $component-spacing--xl;
+}
+
+.jkl-message-box {
+    &__title {
+        margin-bottom: $component-spacing--large;
+    }
+    &__info {
+        @include box($snøhvit);
```

Comment:
This generates some redundant rules: Only `background-color` changes between the variants, but all properties are repeated.

Reply:
fixed